batch/src/main/scala/uk/gov/ons/addressindex/utils/Mappings.scala

```diff
@@ -12,14 +12,15 @@ object Mappings {
               "index": {
                   "similarity": {
                       "default": {
-                          "type": "classic"
+                          "type": "BM25"
```

Comment:
Is this because it's now the default?

Reply:
Classic is no longer available. Iva looked at BM25 before and whilst there were some benefits the results were not as good as classic at the time. Will have to run all the baselines and see...

Reply:
This seems to indicate it is: 
https://www.elastic.co/guide/en/elasticsearch/reference/7.3/similarity.html

Reply:
See also https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-similarity.html for other options if BM25 is NBG.

Reply:
We'll have to wait and see the scores on the doors.